### ISSUE TITLE
Overflow on column width

### ISSUE THREAD

**Issue by colino** [AllenFang/react-bootstrap-table#104]
Hi,
i have a table with a lot of columns. Content of cells seems overwrite others cell's content. What's wrong?

![ss2](https://cloud.githubusercontent.com/assets/4049806/10795476/408e0208-7d9a-11e5-8924-8299ccc6cf33.png)

**Comment by AllenFang** [comment]
I'll fix this problem in near future :)

**Comment by tbaeg** [comment]
+1

**Comment by iamyardem** [comment]
As suggestion, is it possible to add horizontal table scroll in scope of this issue?
It is similar to this: http://allenfang.github.io/react-bootstrap-table/example.html#table-scroll

**Comment by AllenFang** [comment]
I've solved this problem, but bring some side effects, i'm working on that, hope to fix this bug in next version :)

**Comment by tbaeg** [comment]
Also, a simple fix could be.

```
th > div {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}

th > div:hover {
  overflow: visible;
}
```

Although, I think the proper solution would be to have x-axis scrolling as mentioned by @colino. Let me know if I can help in any way @AllenFang.

**Comment by AllenFang** [comment]
@tbaeg, thanks, I've solve all the problem about this issues, it's now on testing. The Actually problem is should not give the <code>fixed</code> to <code>table-layout</code>. In addition, I also fix some border and table height and scroll issues. Anyway, in next version, <code>react-bootstrap-table</code> will auto  horizontal scroll bar if it is a large column table.

**Comment by tbaeg** [comment]
@AllenFang Very cool, I look forward to seeing the results as I could definitely use this feature as well.

**Comment by AllenFang** [comment]
Fix it in v1.2.8 :)
